packages/quantic/force-app/main/default/lwc/quanticTabBar/quanticTabBar.html

```diff
@@ -4,21 +4,28 @@
       <slot></slot>
       <div
         class="tab-bar_more-button slds-tabs_default__item slds-tabs_default__overflow-button slds-is-absolute slds-p-right_none slds-p-left_none"
-        title={labels.moreTabs} role="presentation">
+        title={labels.moreTabs}
+        role="presentation"
+      >
         <div class={dropdownClasses}>
-          <button class="slds-button slds-var-p-horizontal_small" aria-haspopup="true"
-            onclick={toggleDropdown}>{moreButtonLabel}
-            <lightning-icon class={moreButtonIconClasses} icon-name={arrowIconName} size="xx-small" aria-hidden="true">
-            </lightning-icon>
+          <button class="slds-button slds-var-p-horizontal_small" aria-haspopup="true" onclick={toggleDropdown}>
```

Comment:
I would copy the [menu button pattern from the ARIA Authoring Practices Guide](https://www.w3.org/WAI/ARIA/apg/example-index/menu-button/menu-button-links.html).

Reply:
For this PR, `menu` and `menuitem` are reverted back in a similar layout with [menu button pattern from the ARIA Authoring Practices Guide](https://www.w3.org/WAI/ARIA/apg/example-index/menu-button/menu-button-links.html). However `aria-controls` is not implemented since it involves assigning ids to dom elements.